contributing.md

```diff
@@ -286,23 +286,25 @@ Accept the SDK licences:
 $ANDROID_SDK_ROOT/tools/bin/sdkmanager --licenses
 ```
 
-Download an Android system image. Note that you may get a warning about a `.cfg` file not being present.
-You're safe to ignore this.
+Download Android system images for the minimum and targeted API levels. Note that you may get a warning about a `.cfg` file not being present.You're safe to ignore this.
```

Comment:
We need a space after first sentence. i.e. `present. You're safe to ignore this.`